source/api-integration.html.erb

```diff
@@ -108,14 +63,14 @@ title: Integrating your service with GOV.UK Pay
         <p class="govuk-body">You’ll need some developer help to get GOV.UK Pay set up.</p>
         <p class="govuk-body">To get started you can:</p>
         <ul class="govuk-list govuk-list--bullet">
-          <li>read our step by step guidance in our
+          <li>read our step-by-step guidance in our
```

Comment:
I'd be reluctant to add in hyphens/punctuation like this without checking with a content person - that goes for all the extra commas, quote marks etc further down in this commit.

Reply:
Reverted